app/assets/stylesheets/_home_cards.css.scss

```diff
@@ -0,0 +1,75 @@
+/* Home cards */
+.grouped-list-none {
+  padding: 0;
+  text-align: center;
+}
+.list-group-item,
+.list-group-item:hover,
+.list-group-item:focus {
+  -moz-border-radius: 3px;
+  -webkit-border-radius: 3px;
+  border-radius: 3px;
+  -moz-background-clip: padding;
+  -webkit-background-clip: padding-box;
+  background-clip: padding-box;
+  background-color: #fff;
```

Comment:
Color literals like `#fff` should only be used in variable declarations; they should be referred to via variable everywhere else.